kafka/config.go

```diff
@@ -33,6 +45,7 @@ type Config struct {
 	Password  string         `json:"password"`
 	CAPEMFile string         `json:"ca_pem_file"`
 	LogLevel  string         `json:"log_level" split_words:"true"`
+	Timeout   time.Duration  `json:"timeout"`
```

Comment:
The field name `Timeout` is not clear enough. What timeout is referring to exactly? When configuring a Kafka Consumer or Producer you can set tons of them for several purposes. Can you rename the field for making the purpose obvious?